apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -67,10 +67,25 @@ const useSections = section => {
         return section;
       }
     });
+
+    setSections(newSections);
+  };
+
+  const batchUpdateSection = (sectionIdx, updateList) => {
+    const newSections = sections.map((section, idx) => {
+      if (idx === sectionIdx) {
+        let newSection = section;
+        for (const [key, value] of Object.entries(updateList)) {
+          newSection = {...newSection, [key]: value};
```

Comment:
optional nit, can we just do this?
```suggestion
          newSection = {...newSection, ...updateList};
```